app/models/supplejack_api/concerns/privacyable.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module SupplejackApi
+  module Concerns
+    module Privacyable
```

Comment:
Optional point and wouldn't often suggest it, but a concern is often a good candidate for a class doc. Just a sentence that explains what behaviour it's collecting together is usually quite nice. The name usually gives the gist, but a little context goes a long way too.

e.g.
> Fields, validations, scopes and callbacks relating to the private/public acess and hiding access of UserSets.

Reply:
All done :)